sdk/client-core/core-http-okhttp/src/main/java/module-info.java

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module io.clientcore.core.http.okhttp {
+    requires transitive io.clientcore.core;
+
+    requires okhttp3;
+    requires okio;
+    requires kotlin.stdlib;
```

Comment:
can we `requires transitive okhttp3` to get the next two requires?

Reply:
I tried, apparently we do need both.